cmd/kubectl-gadget/utils/trace.go

```diff
@@ -441,15 +442,16 @@ Loop:
 }
 
 // getTraceWatcher returns a watcher on trace(s) whom ID was given as parameter.
-// This watcher can then be used to wait on State.Output modification.
-func getTraceWatcher(traceID string) (watch.Interface, error) {
+// This watcher will wait from version resourceVersion (if set).
```

Comment:
I'm not sure what you are trying to say here... What does "wait from version" mean? I think you might want to modify the other part of the sentence instead:

// getTraceWatcher returns a watcher on trace(s) for the received ID, 
// that have at least the received resourceVersion.
// This watcher can then be used to wait until the State.Output is modified.